src/trans_rdma.c

```diff
@@ -2485,5 +2488,8 @@ uint16_t msk_get_dst_port(struct msk_trans *trans) {
 	return rdma_get_dst_port(trans->cm_id);
 }
 
+const char *msk_wc_status_str(enum ibv_wc_status status) {
```

Comment:
I'd define this as static inline in the header, or actually either make it take msk_data_t as argument or skip it altogether.
Since we're not masking what that struct contains and literally use ibv_wc_status there is no advantage to using this function instead of ibv_wc_status_str directly: if you want to encourage people to look at ibv_wc_status it's useless, and if you want to mask it some coating makes sense to me?

Reply:
I implemented it first as a static inline, but then I thought that this may force the user to link its application with libibverbs explicitly as the ibv_wc_status_str() symbol would be exposed to him/her.

Reply:
Ok, that makes sense, even if I don't think we really care (since we use the structs in mooshika headers I would strongly advise to recompile any mooshika users when libibverbs gets a major update, even if it's not strictly necessary as they usually take care to not modify these structs)

I finally found how to give Quentin access, it's his turn now :)